Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -354,13 +524,53 @@ bool isFunctionJson(string fullName)
             }
         }
 
-        private async Task<string> DeployZipLocally(ZipDeploymentInfo zipDeploymentInfo, ITracer tracer)
+        private async Task ArtifactFetch(IRepository repository, DeploymentInfoBase deploymentInfo, string targetBranch, ILogger logger, ITracer tracer)
+        {
+            var artifactDeploymentInfo = (ArtifactDeploymentInfo)deploymentInfo;
+            //Fetches content from URL if ArtifactURL exists
```

Comment:
Mention that if the ArtifactURL property is defined, we assume that the artifact is to be fetch from that URL. Otherwise, we assume that the artifact is to be read from the request's body.